src/openrct2/actions/LandSetHeightAction.cpp

```diff
@@ -360,7 +360,7 @@ money32 LandSetHeightAction::GetSurfaceHeightChangeCost(SurfaceElement* surfaceE
     {
         int32_t cornerHeight = tile_element_get_corner_height(surfaceElement, i);
         cornerHeight -= map_get_corner_height(_height, _style & TILE_ELEMENT_SURFACE_SLOPE_MASK, i);
-        cost += MONEY(abs(cornerHeight) * 5 / 2, 0);
+        cost += 2.5_GBP * cornerHeight;
```

Comment:
This looks wrong. What if cornerHeight is negative? Seems that make for a negative cost.

Reply:
Nice catch, didn't mean to remove the `abs`.